### PR TITLE
Adjust explanation of single character mode.

### LINKED ISSUE #7777: Class specific items for alts do not appear in vault when displaying only current character of different class
### DIM Version

Version 7.1.0 (release), built on 1/16/2022, 9:12:32 PM

### Browser Details

Chrome Version 97.0.4692.71

### OS Details

OSX Monterey

### Describe the bug

`is:sword is:void` returns 5 items, but only 3 items are displayed in the vault and the other two are hidden. (crown splitter and quick fang hidden, I am on a warlock)

image
Expanding to view all three characters shows the items in the vault. 

image

The behavior I would expect is to see them in the vault, but not equippable.

### Logs

_No response_

### PR DESCRIPTION
**Pull request by bhollis**
Fixes #7777

### CODE REVIEW

File: config/i18n.json [hunk 1]
```diff
@@ -928,7 +928,7 @@
     "ShowNewItems": "Show a red dot on new items",
     "ExpandSingleCharacter": "Show all characters",
     "SingleCharacter": "Only show current character",
-    "SingleCharacterExplanation": "DIM will hide all but the most recently played character. Those characters' inventories will appear as if they were in the vault.",
+    "SingleCharacterExplanation": "DIM will hide all but the most recently played character. Items from hidden characters' inventories that can still be used by the current character will appear as if they were in the vault, even though they are still on the hidden characters. Class-specific items that can't be used by the current character will be hidden entirely.",
```
Comment (nev-r) [inline config/i18n.json:931]:
```suggestion
    "SingleCharacterExplanation": "DIM will show only the most recently played character.\nItems held by hidden characters will appear in the vault, if they can be used by the current character.\nItems specific to other classes will be hidden entirely.",
```

File: config/i18n.json [hunk 2]
```diff
@@ -928,7 +928,7 @@
     "ShowNewItems": "Show a red dot on new items",
     "ExpandSingleCharacter": "Show all characters",
     "SingleCharacter": "Only show current character",
```
Comment (nev-r) [inline config/i18n.json:930]:
```suggestion
    "SingleCharacter": "Single-Character Mode",
```
i'd be fine with promoting "Single-Character Mode" to the actual name of the feature.
the current separateness and colon structure don't really make sense.

**Review by nev-r — COMMENTED** [review]
tried to both pare down the text, and give it room to breathe. would love to see this bumped up to a full PressTip instead of a weak title attribute

File: config/i18n.json (hunk 2 above)
Comment (bhollis) [reply config/i18n.json:930]: I could go either way. Single character mode maybe could be interpreted in some weird way.

Comment (nev-r) [reply config/i18n.json:930]: Single-Character View?

Comment (bhollis) [reply config/i18n.json:930]: Yeah that's clearer

**Pull request merged**